README.md

```diff
@@ -1,39 +1,57 @@
 # Assessing your JavaScript Fundamentals
-This sprint you worked really hard on learning the fundamentals of JavaScript! This sprint challenge will touch on each concept covered throughout this week.  Follow each task in order but feel free to skip something if you're struggling and come back to it later.
 
-* This project is to be worked on alone but you can use outside resources. You can _reference_ any old code you may have and the training kit content.
+This sprint you worked really hard on learning the fundamentals of JavaScript! This sprint challenge will touch on each concept covered throughout this week. Follow each task in order but feel free to skip something if you're struggling and come back to it later.
+
+- This project is to be worked on alone but you can use outside resources. You can _reference_ any old code you may have and the training kit content.
 
 ## Task 1: Set Up The Project With Git
 
-* [ ] Fork the project into your GitHub user account
-* [ ] Clone the forked project into a directory on your machine
-* [ ] You are now ready to build this project with your preferred IDE
+- [x] Fork the project into your GitHub user account
+- [x] Clone the forked project into a directory on your machine
+- [x] You are now ready to build this project with your preferred IDE
 
 ## Task 2: Comprehension Questions
-Answer the following questions to the best of your ability. You can exercise your Googling skills and use training kit.  Open up the [Answers.md](Answers.md) file and record your responses there.
 
-1. Describe the biggest difference between `.forEach` & `.map`.
-2. What is the difference between a function and a method?
-3. What is closure?
+Answer the following questions to the best of your ability. You can exercise your Googling skills and use training kit. Open up the [Answers.md](Answers.md) file and record your responses there.
+
+1. Describe the biggest difference between `.forEach` & `.map`: forEach applies to every item in an array and doesn't return anything, map returns an array of the same size.
+
+2. What is the difference between a function and a method? A function is used to return a value (IE string, number, etc). A method is like a function, but it's an internal part of a class.
+
+3. What is closure? Closure is an inner function that has access to the outer
+
 4. Describe the four rules of the 'this' keyword.
```

Comment:
Perfect! Keep these in mind as everything else will build on these concepts. THIS, while being ridiculous in it's own way, will definitely make things easier to understand later on.